plugins/volumeFilter.go

```diff
@@ -163,23 +163,23 @@ func volumeFilterFn(dailyOTB *VolumeFilterConfig, dailyTBBAccumulator *VolumeFil
 		return nil, fmt.Errorf("error when running the isSelling check for offer '%+v': %s", *op, e)
 	}
 
-	sellPrice, e := strconv.ParseFloat(op.Price, 64)
+	offerPrice, e := strconv.ParseFloat(op.Price, 64)
 	if e != nil {
 		return nil, fmt.Errorf("could not convert price (%s) to float: %s", op.Price, e)
 	}
 
-	amountValueUnitsBeingSold, e := strconv.ParseFloat(op.Amount, 64)
+	offerAmountValueUnits, e := strconv.ParseFloat(op.Amount, 64)
```

Comment:
@debnil this variable name change has introduced unwanted ambiguity..

`amountValueUnitsBeingSold` tells us that the value corresponds to the selling asset in the manageSellOffer.
`offerAmountValueUnits` does not tell us whether the amount corresponds to the selling asset or buying asset in the manageSellOffer.

1. need to figure out what this variable means when we use `buy` types, which is not super-straightforward -- may need to take a TDD approach or intrusive debugging to figure this out.
2. need to name this variable accordingly so it is super clear.

the reason we need to do the two steps above is because we unfortunately had solved the trivial case previously -- doing a "sell" type operation using a manageSellOffer where amount and price all follow the selling asset (because that's how manageSellOffer works). the hard part is to now figure out what values we will see when we invert it.

let's discuss in our 1-1 session tomorrow (Thursday) about what you end up finding from your debugging.

Reply:
Ah, got it, ok. I'll revert these names as well.